### ISSUE TITLE
Three Track: Chord mode doesn't work as expected.

### ISSUE THREAD

**Issue by Deadairx** [patheros/JPManuals#9]
Using the Three Track module, enabling Chord mode seems to change the module to a single sequence instead of 3 sequences that have the same rhythm. The UI shows "Chord" but the right-click menu still has a check next to "Chord Mode: Off"

Version: `2.0.2`
VCV Rack Pro `2.4.1`
OS: `macOS ARM64`

![2023-10-31 10 27 10](https://github.com/patheros/JPManuals/assets/12243529/172c0351-116c-40b7-bcf6-472159a8c167)

**Comment by patheros** [comment]
Good catch. I should have a fix out for this in the next update. It appears to be mostly cosmetic as the module functions as normal for me but there is a a chance this could cause a crash. If it does, setting the module to Chord mode by right clicking the A/B/C display on the panel, and selecting "Chord" shouldn't crash.

**Comment by patheros** [comment]
Was fixed with version 2.0.1
